backend/src/main/java/io/metersphere/track/service/TestPlanReportService.java

```diff
@@ -497,16 +493,16 @@ public TestPlanReport finishedTestPlanReport(String testPlanReportId, String sta
                             }
                         }
                         content.setPlanApiCaseReportStruct(JSONObject.toJSONString(apiTestCases));
-                    }catch (Exception e){
-                        LogUtil.error("update test plan api error! "+e.getMessage());
+                    } catch (Exception e) {
+                        LogUtil.error("update test plan api error! " + e.getMessage());
```

Comment:
maybe this is better： LogUtil.error("update test plan api error! " , e.getMessage());